apps/qubit/modules/informationobject/templates/_cardViewResults.php

```diff
@@ -11,15 +11,15 @@
     <?php endif; ?>
 
       <a href="<?php echo url_for(array('module' => 'informationobject', 'slug' => $doc['slug'])) ?>">
-        <?php if (!empty($doc['digitalObject']['thumbnailPath'])
+        <?php if (isset($doc['digitalObject']) && !empty($doc['digitalObject']['thumbnailPath'])
```

Comment:
Ugh, right, `sfOutputEscaperArrayDecorator`... that's sad but nothing we can do!